config/config.go

```diff
@@ -0,0 +1,4 @@
+package config
```

Comment:
I think it's great to have a config package, but you would usually like to read these values from a file instead of just having them hardcoded, this is because these values can potentially change betweens the different environments the app could be living in

Reply:
Also, you don't want to have sensitive information being versioned, like DB connection details and such